projects/01-rubygems-cli/solution/spec/lib/search_command_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+require "./lib/search_command"
+
+RSpec.describe SearchCommand do
+  subject(:program) { described_class }
+
+  describe ".execute" do
+    subject(:execute) { program.execute(arg) }
+
+    context "with valid keyword" do
+      let(:arg) { "rspec" }
+
+      it "returns SearchCommandResult" do
+        result = execute
+
+        expect(result.class).to be SearchCommandResult
```

Comment:
There is a debate here :)

Rule of thumb (IMO) is that one should be testing the public interface of their.
Here it depends what you define your public interface, is it the CLI interface, or the command itself.